app/src/test/java/pjwstk/receipemate/app/viewfactory/category/CategoryViewFactoryUnitTest.java

```diff
@@ -0,0 +1,78 @@
+package pjwstk.receipemate.app.viewfactory.category;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import pjwstk.receipemate.app.entity.CategoryImage;
+import pjwstk.receipemate.app.view.category.CategoryImageView;
+import pjwstk.receipemate.app.view.category.CategoryView;
+import pjwstk.receipemate.app.entity.Category;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+
+public class CategoryViewFactoryUnitTest {
+
+    private CategoryViewFactory categoryViewFactory;
+    private Category category;
+    private CategoryImageView expectedCategoryImageView;
+
+    @BeforeEach
+    public void setup() {
+        //given
+        CategoryImageViewFactory mockCategoryImageViewFactory = mock(CategoryImageViewFactory.class);
+        categoryViewFactory = new CategoryViewFactory(mockCategoryImageViewFactory);
+
+        //Initialize Category
+        category = new Category();
+
+        category.setId(1);
+        category.setName("Test Name");
+        CategoryImage categoryImage = new CategoryImage();
+        category.setCategoryImage(categoryImage);
+
+        //Mock mapping CategoryImage to CategoryImageView
+        expectedCategoryImageView = new CategoryImageView();
+        when(mockCategoryImageViewFactory.make(categoryImage)).thenReturn(expectedCategoryImageView);
+    }
+
+    @Test
+    void make_ShouldMapCategoryToCategoryView() {
```

Comment:
Postaraj się ujednolicić nazwy testów, chyba że zostajemy przy twoim nazewnictwie, chodzi o to żeby projekt był jak najbardziej ujednolicony

Reply:
ok, usunałem po prostu. dajać nazwe metody przed, było dla mnie czytelniejsze. :)